### PR TITLE
Added summaryLastDay and summaryToday

### PR DESCRIPTION
**Pull request by diegodurrod**

### CODE REVIEW

File: src/Traits/ReportUtilityTrait.php [hunk 1]
```diff
@@ -83,4 +83,30 @@ public function summaryLastYear(array $data = array())
         return $this->summary( $data );
     }
 
-}
\ No newline at end of file
+    public function summaryLastDay(array $data = array())
```
Comment (elimentz) [inline src/Traits/ReportUtilityTrait.php]: Can you rename this to `summaryYesterday`? I understand you're trying to be consistent but "last day" isn't really intuitive :-)

Comment (diegodurrod) [reply src/Traits/ReportUtilityTrait.php]: No problem ;-)

**Comment by diegodurrod** [comment]
Method name has been renamed in the latest commit. If you consider it, you can accept the PR.

**Comment by elimentz** [comment]
Thanks for contributing!

**Comment by elimentz** [comment]
Available in version 0.8.0

**Pull request merged**